inginious/frontend/flask/mongo_sessions.py

```diff
@@ -67,9 +67,13 @@ def _get_signer(self, app):
     def open_session(self, app, request):
         # Check for cookieless session in the path
         path_session = re.match(r"(/@)([a-f0-9A-F_]*)(@)", request.path)
+        is_lti_launch = re.match(r"(/lti)", request.path)
```

Comment:
`request.path.startswith("/lti")` ?

Reply:
or you meant `re.match(r"(/lti/?)", request.path)` maybe?

You should ensure that it's either a path under the /lti/ path of the /lti page:
`request.path == "/lti" or request.path.startswith("/lti/")` is another non-regex equivalent, in order to avoid "using a canon to kill a fly", like we say in french ;-)